packages/hub/src/lib/parse-safetensors-metadata.ts

```diff
@@ -15,7 +15,7 @@ export const SAFETENSORS_INDEX_FILE = "model.safetensors.index.json";
 export const RE_SAFETENSORS_FILE = /\.safetensors$/;
 export const RE_SAFETENSORS_INDEX_FILE = /\.safetensors\.index\.json$/;
 export const RE_SAFETENSORS_SHARD_FILE = /\d{5}-of-\d{5}\.safetensors$/;
-const PARALLEL_DOWNLOADS = 5;
```

Comment:
was there any reasons why it was 5? this PR bumps it up to 40. Would we hit any rate-limit error? Also, all the requests are [http range requets](https://developer.mozilla.org/en-US/docs/Web/HTTP/Range_requests) for the header to get metadata

cc: @coyotte508 @SBrandeis